map/src/menu/travel/TravelMenu.jsx

```diff
@@ -0,0 +1,217 @@
+import headerStyles from '../trackfavmenu.module.css';
+import { AppBar, Box, Button, CircularProgress, IconButton, Toolbar, Typography } from '@mui/material';
+import React, { useContext, useEffect, useMemo, useState } from 'react';
+import { ReactComponent as CloseIcon } from '../../assets/icons/ic_action_close.svg';
+import { ReactComponent as SortDateIcon } from '../../assets/icons/ic_action_sort_by_date.svg';
+import { MENU_INFO_CLOSE_SIZE } from '../../manager/GlobalManager';
+import AppContext from '../../context/AppContext';
+import activities from '../../resources/activities.json';
+import {
+    DEFAULT_TAG_ICON_COLOR,
+    DEFAULT_TAG_ICON_SIZE,
+    getIcon,
+    getSvgIcon,
+} from '../../infoblock/components/wpt/WptTagsProvider';
+import styles from './travel.module.css';
+import buttonStyles from '../login/login.module.css';
+import CustomSelect from './CustomSelect';
+import { useTranslation } from 'react-i18next';
+import EmptyTravel from '../errors/EmptyTravel';
+import EmptyLogin from '../login/EmptyLogin';
+import TravelRoutesResult from './TravelRoutesResult';
+import capitalize from 'lodash/capitalize';
+
+export const ALL_YEARS = 'all';
+
+export default function TravelMenu() {
+    const ctx = useContext(AppContext);
+    const { t } = useTranslation();
+
+    const DEFAULT_ACTIVITY = 'hiking';
+    const DEFAULT_YEAR = new Date().getFullYear();
+
+    const [selectedActivityType, setSelectedActivityType] = useState(DEFAULT_ACTIVITY);
+    const [selectedYear, setSelectedYear] = useState(DEFAULT_YEAR);
+    const [updatedActivities, setUpdatedActivities] = useState([]);
+    const [travelResult, setTravelResult] = useState(null);
+    const [loadingResult, setLoadingResult] = useState(false);
+
+    useEffect(() => {
+        if (ctx.searchTravelRoutes?.res) {
+            setTravelResult(ctx.searchTravelRoutes?.res);
+            setLoadingResult(false);
+        }
+    }, [ctx.searchTravelRoutes?.res]);
+
+    useEffect(() => {
+        ctx.setTravelFilter({
+            activity: selectedActivityType,
+            year: selectedYear,
+        });
+    }, [selectedActivityType, selectedYear]);
+
+    useEffect(() => {
+        if (ctx.travelFilter) {
+            setSelectedActivityType(ctx.travelFilter.activity);
+            setSelectedYear(ctx.travelFilter.year);
+        }
+    }, []);
+
+    // Create years array
+    const years = [
+        { id: ALL_YEARS, label: capitalize(ALL_YEARS) },
+        ...Array.from({ length: 20 }, (_, i) => {
+            const year = new Date().getFullYear() - i;
+            return { id: year, label: `${year}` };
+        }),
+    ];
+
+    // Create activities array
+    const activitiesArr = useMemo(() => {
+        return activities?.groups.reduce((act, group) => {
+            if (act.length === 0) {
+                act.push({
+                    id: 'osmand',
+                    label: 'Osmand',
+                    type: 'osmand',
```

Comment:
osmand is not a kind of activity anymore

possibly we should add `nospeed` instead of